package.json

```diff
@@ -24,9 +24,9 @@
     "build-strict-log": "npm run build:strict > build-debug.log 2>&1 && tail -3 build-debug.log",
     "build:compiled": "gulp buildCompiled",
     "build:compressed": "npm run build:compiled",
-    "build:deps": "gulp buildDeps",
+    "build:deps": "gulp buildJavaScriptAndDeps",
+    "build:js": "gulp buildJavaScript",
```

Comment:
Is this just for testing? If so, is it necessary to have it be part of the `package.json`?